tree/treeplayer/inc/ROOT/TArrowDS.hxx

```diff
@@ -0,0 +1,202 @@
+#ifndef ROOT_TARROWTDS
+#define ROOT_TARROWTDS
+
+#include <arrow/table.h>
+#include <memory>
+
+#include "ROOT/TDataFrame.hxx"
+#include "ROOT/TDataSource.hxx"
+
+namespace ROOT {
+namespace Experimental {
+namespace TDF {
+
+class TArrowDS final : public ROOT::Experimental::TDF::TDataSource {
```

Comment:
This is what is done in `TCsvDS` already. I will fix it, but I think that this kind of comments should be enforced everywhere the same way via a static analyser in the CI if they are considered a problem. It's hard otherwise to contribute if we cannot rely on the fact that code in the repository is actually not conforming to agreed practices.

Reply:
I'm sorry. I don't mean to say what you are doing is wrong or not conforming. It's just that this was found to be a problem on Windows by @bellenot, so we have to at least keep it in mind here and fix (and the other places too).